### PR TITLE
added slack chat, updated machine_spec, added amin.xml

### PR DESCRIPTION
**Pull request by bdumm2**

### CODE REVIEW

File: lib/Amin/Machine/Machine_Spec/machine_spec.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <machine xmlns:amin="http://projectamin.org/ns/">
 	<!--	LICENSE:
 	        Please see the LICENSE file included with this distribution 
-		or see the following website http://projectamin.org.
 	-->
 	<!--what type of machine is this? -->
 	<name>Amin::Machine::Dispatcher</name>	
-	<!-- <amin:command name="mkdir"/> -->
```
Comment (swishy) [inline lib/Amin/Machine/Machine_Spec/machine_spec.xml:8]: Are we going to be adding such for all filters in machine spec?

Comment (bdumm2) [reply lib/Amin/Machine/Machine_Spec/machine_spec.xml:8]: no i thought i took em all out :)

**Review by swishy — APPROVED** [review]
Looks good to me, wheres the Perl? :)

File: lib/Amin/Machine/Machine_Spec/machine_spec.xml (hunk 1 above)
Comment (swishy) [reply lib/Amin/Machine/Machine_Spec/machine_spec.xml:8]: No worries can clean up next commit :)

**Pull request merged**